tests/integration/update_cluster/minimal/in-v1alpha2.yaml

```diff
@@ -30,6 +30,7 @@ spec:
   nonMasqueradeCIDR: 100.64.0.0/10
   sshAccess:
     - 0.0.0.0/0
+  sshKeyName: ""
```

Comment:
Any reason for this change?
```suggestion
  sshKeyName: ""
```

Reply:
none, it's late here and I messed up while simplifying the new test I added. I just fixed it & the other test I just added.

Reply:
hehe :)